analytical/utils.py

```diff
@@ -163,3 +163,90 @@ class AnalyticalException(Exception):
     be silenced in templates.
     """
     silent_variable_failure = True
+
+def build_paq_cmd(cmd, args=[]):
+    """
+    :Args:
+        - cmd: The command to be pushed to paq (i.e enableHeartbeatTimer or contentInteraction)
+        - args: Arguments to be added to the paq command. This is mainly
+            used when building commands to be used on manual event trigger.
+        
+    :Returns:
+        A complete '_paq.push([])' command in string form
+    """
+    def __to_js_arg(arg):
+        """
+        Turn 'arg' into its javascript counter-part
+        :Args:
+            - arg: the argument that's to be passed to the array in _paq.push()
+        :Return:
+            The javascript counter-part to the argument that was passed
+        """
+        if isinstance(arg, dict):
+            arg_cpy = deepcopy(arg)
+            for k, v in arg_cpy.items():
+                arg.pop(k)
+                arg[__to_js_arg(k)] = __to_js_arg(v)
+            return arg
+        elif isinstance(arg, bool):
+            if arg:
+                arg = "true"
+            else:
+                arg = "false"
+        elif isinstance(arg, list):
+            for elem_idx in range(len(arg)):
+                arg[elem_idx] = __to_js_arg(arg[elem_idx])
+
+        return arg
+
+    paq = "_paq.push(['%s'" % (cmd)
+    if len(args) > 0:
+        paq += ", "
+        for arg_idx in range(len(args)):
+            current_arg = __to_js_arg(args[arg_idx])
+            no_quotes = type(current_arg) in [bool, int, dict, list]
+            if arg_idx == len(args)-1:
```

Comment:
This is – and the `else` black below – is not an elegant solution. Why don't you simply always add the `", "` and replace it by `"]);"` after the end of the loop? e.g.

```python
   paq = paq[:-3)] + "]);"
   return paq
```